src/uct/ib/mlx5/ib_mlx5_log.c

```diff
@@ -86,9 +86,6 @@ ucs_status_t uct_ib_mlx5_completion_with_err(uct_ib_iface_t *iface,
     case MLX5_CQE_SYNDROME_LOCAL_PROT_ERR:
         snprintf(err_info, sizeof(err_info), "Local protection");
         break;
-    case MLX5_CQE_SYNDROME_WR_FLUSH_ERR:
-        snprintf(err_info, sizeof(err_info), "WR flushed because QP in error state");
-        break;
```

Comment:
maybe move code from 74-76 in this case ? and remove the if?

Reply:
done